platforms/qurt/src/px4_layer/px4_qurt_impl.cpp

```diff
@@ -166,3 +166,9 @@ int fputc(int c, FILE *stream)
 {
 	return c;
 }
+
+int putchar(int character)
+{
+	PX4_ERR("Error: Calling unresolved symbol stub:[%s]", __FUNCTION__);
```

Comment:
@dagar Is is better if we just print nothing for now? The only thing I worry about is that we forget about this and at some point it bites us.